model/metrics/hardware/common.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  # COMMON METRICS
+  - id: metric.hardware.attributes
+    type: attribute_group
+    brief: "Attributes for hardware metrics"
+    extends: hardware.attributes.common
+    attributes:
+      - ref: hw.type
+        requirement_level: required
+
+  - id: metric.hardware.energy
+    type: metric
+    metric_name: hw.energy
+    stability: experimental
+    brief: "Energy consumed by the component"
+    instrument: counter
+    unit: "J"
+    extends: metric.hardware.attributes
+
+  - id: metric.hardware.errors
+    type: metric
+    metric_name: hw.errors
+    stability: experimental
+    brief: "Number of errors encountered by the component"
+    instrument: counter
+    unit: "{errors}"
```

Comment:
```suggestion
    unit: "{error}"
```

Reply:
We used to use plural for non-standard units (in curly brackets). Has this changed?